docs/changelog.rst

```diff
@@ -2,6 +2,8 @@
 Changelog
 =========
 
+* :feature: Update WebUI to version 0.7.1 https://github.com/raiden-network/webui/releases/tag/v0.7.1
```

Comment:
Sorry to bother again, but needs to have a link to an issue. If not it needs `-`. Like this:
 ```
:feature:`-`
```

Reply:
no bother give me a moment and I will fix it

Reply:
Do you think it makes sense opening an issue when updating the WebUI?

Reply:
naaah -- just use `-`.